libraries/AC_Autorotation/AC_Autorotation.h

```diff
@@ -92,8 +184,14 @@ class AC_Autorotation
     } _flags;
 
     //--------Internal Functions--------
+    // set the collective in the motors library
     void set_collective(float _collective_filter_cutoff) const;
 
     // low pass filter for collective trim
     LowPassFilterFloat col_trim_lpf;
+
+    //--------References to Other Libraries--------
+    AP_InertialNav&    _inav;
+    AP_AHRS&           _ahrs;
+    AP_MotorsHeli*     _motors_heli;
```

Comment:
```suggestion
    AP_MotorsHeli*&     _motors_heli;
```
You can refence the copter pointer and set it in the constructor. You will have to check for nullprt before use.